autogen.sh

```diff
@@ -29,7 +29,8 @@ autoreconf --force --install --symlink --warnings=all
 args="\
 --sysconfdir=/etc \
 --localstatedir=/var \
---prefix=/usr"
+--prefix=/usr \
+--disable-valgrind-sgcheck"
```

Comment:
I see that this change will automatically be applied to the OBS builds, since the files in `data/obs-packaging/` call `./autogen.sh`.